workshops/parallax_effect/README.md

```diff
@@ -0,0 +1,180 @@
+}
+```
+
+The `hrparallax` function makes the layer translate along x-direction as you scroll vertically.
+The positive value translates rightwards and negative value translates leftwards. Discover more about `translateX()` [here](https://developer.mozilla.org/en-US/docs/web/css/transform-function/translateX).
+
+<a href="https://cloud-7jtog4dbv.vercel.app/screenshot_2020-09-22_185155.png"><img src="https://cloud-7jtog4dbv.vercel.app/screenshot_2020-09-22_185155.png" alt="translateX() example" width="auto" height="200px"/></a>
+
+Add the following code
+
+```javascript
+hrparallax('.l4', window.scrollY, -0.5)
+hrparallax('.l5', window.scrollY, 0.5)
+```
+
+For rightward movement the `speed` will be negative and for leftward movement it should be positive. Here `l4` moves rightwards and `l5` moves leftwards.
+
+Then your final page looks something [like this](https://total-parallax.giridharhackclu.repl.co/).
+
+![Final Result](https://cloud-1z03ndewk.vercel.app/ezgif.com-gif-maker__2_.gif)
+
+That's it! You added both `vertical-parallax` and `horizontal-parallax` effects. Well done.
+
+## Hacking
+
+You can create different parallax effects.
+
+1. In this workshop, I focused only on scrolling vertically. You can use <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollX">`scrollX`</a> for creating parallax while scrolling horizontally.
+1. You can use <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate">`transform: translate(X,Y);`</a> or <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate3d">`transform: translate3d(X,Y,Z);`</a> for creating different effects.
+
+You can add parallax-effect to your websites for creating better user experience and display your content more interactively.
+
+Make use of your creativity and make different parallax effects. Share it with me on slack [@giridhar](https://hackclub.slack.com/team/U013E6KE9UJ), I'd love to here from you!
+
+## Inspiration
+
+These are some projects made using parallax-effect.
+
+- Example-1 [demo](https://exampleone.giridharhackclu.repl.co/). [source code](https://repl.it/@Giridharhackclu/exampleone#index.html).
+  Real application of parallax effect.
+- Example-2 [demo](https://exampletwo.giridharhackclu.repl.co/). [source code](https://repl.it/@Giridharhackclu/exampletwo#index.html).
+  _Zoom on scroll_ - In this example, width of elements changes as you scroll.
+- Example-3 [demo](https://horizontal-parallax-effect.giridharhackclu.repl.co/). [source code](https://repl.it/@Giridharhackclu/horizontal-parallax-effect#index.html).
+  _Mouse-move parallax_ - As you move mouse-pointer, the `innerHeight` and the `innerWidth` of the elements changes.
+
+Check this article for awesome next-level parallax examples - [article](https://www.awwwards.com/30-great-websites-with-parallax-scrolling.html)
+
+Hope you love this workshop! :v:
+
+## Speed-1
```

Comment:
Any reason why this is at the very end? It feels like a part of the workshop got bitten off and spit out at the end. Should this be higher up?

Reply:
I got inspired this form sam's workshop
He asked a question and linked the answer at the last
so I did in the same way
No problem, I will change it!